infrastructure/instances.tf

```diff
@@ -186,10 +161,10 @@ output "nomad_server_2_ip" {
 resource "aws_instance" "nomad_server_3" {
   ami = "${data.aws_ami.ubuntu.id}"
   instance_type = "t2.small"
-  availability_zone = "${var.region}b"
+  availability_zone = "${var.region}a"
   vpc_security_group_ids = ["${aws_security_group.data_refinery_worker.id}"]
-  iam_instance_profile = "${aws_iam_instance_profile.ecs_instance_profile.name}"
-  subnet_id = "${aws_subnet.data_refinery_1b.id}"
+  iam_instance_profile = "${aws_iam_instance_profile.data_refinery_instance_profile.name}"
+  subnet_id = "${aws_subnet.data_refinery_1a.id}"
```

Comment:
We are still only operating in a single subnet of a single region.

Reply:
Isn't this intentional for now to make debugging easier?